src/Jira/Api.php

```diff
@@ -741,16 +741,15 @@ public function api(
 				}
 			}
 
-			if ( $return_as_array ) {
+			if ($return_as_array) {
 				return $json;
 			}
-			else {
-				return new Result($json);
-			}
-		}
-		else {
-			return false;
+
+			return new Result($json);
+
```

Comment:
Please remove an empty line.

---

No need to add an empty line before closing `}`.